insights/specs/default.py

```diff
@@ -435,6 +435,7 @@ def semid(broker):
                            "/etc/opt/rh/rh-nginx*/nginx/nginx.conf"
                            ])
     nmcli_dev_show = simple_command("/usr/bin/nmcli dev show")
+    nmcli_conn_show = simple_command("/usr/bin/nmcli conn show")
```

Comment:
Please move this spec ahead of the nmcli_dev_show for alphabet order.

Reply:
acknowledged, updated in the code in latest PR